src/com/company/reverse/Reverse.java

```diff
@@ -0,0 +1,21 @@
+package com.company.reverse;
+
+public class Reverse {
```

Comment:
It’s suggested that each class should have only one responsibility.
https://en.wikipedia.org/wiki/Single-responsibility_principle

Reply:
Done